### ISSUE TITLE
Alt text label not showing for gifs or videos

### ISSUE THREAD

**Issue by egel557** [sk22/megalodon#409]
**Describe the bug**

There is no label indicating whether a gif or a video has alt text or not the way that there is for regular images. Because of this, I often mistakenly boost a post that doesn't have alt text.

**Version**

Megalodon version: v1.1.5+fork.74
